aisera/manifest.json

```diff
@@ -0,0 +1,45 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "ed21d816-fc99-4454-99d9-da6404d448a7",
+  "app_id": "aisera_aiops",
+   "assets": {
+    "integration": {
+      "source_type_name": "aisera",
+      "configuration": {}
+    }
+  },
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Aisera Aiops.",
+    "title": "Aisera",
+     "media": [
+      {
+        "media_type": "image",
+        "caption": "Aisera AiOps",
+        "image_url": "logos/aisera_logo.png"
+      }
```

Comment:
You do not need to include the logo here. Instead, please upload screenshots of your platform and Datadog dashboard that follow our [media guidelines](https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#media-carousel).